### ISSUE TITLE
Evaluation data from table 1

### ISSUE THREAD

**Issue by BloodLemonS** [MCG-NKU/E2FGVI#81]
How to obtain evaluation data for table 1? The values obtained when testing the DAVIS data of the release model using the provided evaluation.py file differ significantly from those in the paper.

**Comment by Paper99** [comment]
Just follow our instructions to perform the evaluation.
You may need to check the installed requirements.

**Comment by BloodLemonS** [comment]
I have checked the installation requirements and found that installing different versions of third-party libraries due to differences in device hardware configuration may result in significant calculation errors?

**Comment by Paper99** [comment]
We have tested on both 3090 and V100, and the metrics are consistent. Could you please tell me whether you used our mask for evaluation?

**Comment by BloodLemonS** [comment]
Yes, we conducted the test on the 3070 graphics card using 50 provided test datasets. Except for one dataset with an inf test result, the average PSNR of the other datasets was around 20

**Comment by Paper99** [comment]
This is very weird. Please carefully check whether the environment configuration is the same as [ours](https://github.com/MCG-NKU/E2FGVI/blob/master/environment.yml). The difference in GPU will not lead to different results.
